src/dbnode/persist/fs/commitlog/commit_log.go

```diff
@@ -184,6 +228,34 @@ func (l *commitLog) Open() error {
 	return nil
 }
 
+func (l *commitLog) ActiveLogs() ([]File, error) {
+	l.closedState.Lock()
+	defer l.closedState.Unlock()
```

Comment:
nit: This just needs to be RLock() and RUnlock() yeah?  Don't see it changing the closed state as far as I can tell?

Reply:
done